istioctl/cmd/auth.go

```diff
@@ -61,8 +61,11 @@ THIS COMMAND IS STILL UNDER ACTIVE DEVELOPMENT AND NOT READY FOR PRODUCTION USE.
 
   # Check the TLS/JWT/RBAC policy status from a config dump file:
   istioctl experimental auth check -f httpbin_config_dump.txt`,
-		Args: cobra.MaximumNArgs(1),
 		RunE: func(cmd *cobra.Command, args []string) error {
+			if len(args) > 1 {
```

Comment:
Put this in a function in the `Args` field. That way it automatically prints usage.